server/routerlicious/packages/services-shared/src/redisSocketIoAdapter.ts

```diff
@@ -335,6 +336,15 @@ export class RedisSocketIoAdapter extends Adapter {
                 rooms: new Set([room]),
             };
 
+            if (packet.data?.length > 1) {
+                const msgList = packet.data[2];
```

Comment:
Maybe we can add a comment here explaining what `data[]` represents and why we are getting data[2] specifically?

Also, is there any chance we could have data[] only having fewer elements causing an error when trying to access index number 2?